page-objects/pages/coursePages/CRSSAddCoursePage.js

```diff
@@ -1 +1,224 @@
+		const container = await driver.$(containerSelector);
+		const listOfElements = await container.$$(chosenDefInput);
+		const valueList = await eh.getValueListFromListOfElements(listOfElements);
+		const isOnlyOneText = valueList.length == 1;
+		await expect(isOnlyOneText).is.equal(true);
+		await expect(valueList).includes(defaultText);
+	},
+
+	sectionIsDisplayed: async function (sectionNumber) {
+		const sectionToCheck =
+			sectionNumber == 1 ? sectionNumber : sectionNumber - 1;
+		const selector = this.getSectionSelector(sectionToCheck);
+		const element = await driver.$(selector);
+		const hasChildren = (await element.$$(".//*")).length > 0;
+
+		if (sectionNumber == 1) {
+			await expect(await eh.isElementPresent(selector)).to.equal(true);
+			await expect(hasChildren).to.equal(false);
+		} else {
+			await expect(hasChildren).to.equal(true);
+		}
+	},
+
+	sectionIsNotDisplayed: async function (sectionNumber) {
+		const sectionToCheck =
+			sectionNumber == 1 ? sectionNumber : sectionNumber - 1;
+		const element = await driver.$(this.getSectionSelector(sectionToCheck));
+		const hasChildren = (await element.$$(".//*").length) > 0;
+
+		if (sectionNumber == 1) {
+			await expect(await eh.isElementPresent(sectionToCheck)).to.equal(
+				false
+			);
+		} else {
+			await expect(hasChildren).to.equal(false);
+		}
+	},
+
+	getSectionSelector: function (sectionNumber) {
+		let selector;
+		switch (sectionNumber) {
+			case 1:
+				selector = section.one;
+				break;
+			case 2:
+				selector = section.two;
+				break;
+			case 3:
+				selector = section.three;
+				break;
+			default:
+				console.error(
+					`This section: ${sectionNumber} does not exist on the list of possible choices`
+				);
+				break;
+		}
+		return selector;
+	},
+
+	getUserName: async function () {
+		const cookie = await driver.getCookies(["jwt"]);
+		const jwt = cookie[0].value;
+		const info = await axios.request({
+			url: "http://localhost:3030/me",
+			method: "get",
+			headers: {
+				Authorization: `${jwt}`,
+			},
+		});
+		const firstName = info.data.firstName;
+		const lastName = info.data.lastName;
+		return firstName + " " + lastName;
+	},
+
+	//Course data section
+	courseNameIsNotEntered: async function () {
+		const courseNameContainer = await driver.$(courseNameInput);
+		const placeholderText = await courseNameContainer.getAttribute("placeholder");
+		await expect(placeholderText).to.equal("z.B. Mathe 10a");
+	},
+
+	setCourseName: async function (courseName) {
+		const courseNameContainer = await driver.$(courseNameInput);
+		await courseNameContainer.setValue(courseName);
+	},
+
+	setColour: async function (colourName) {
+		const listOfColours = courseColour;
+		if (listOfColours.includes(colourName)) {
+			const childNumber = listOfColours.indexOf(colourName) + 1;
+			const colour = await driver.$(
+				colourPicker + `:nth-child(${childNumber})`
+			);
+			await colour.click();
+		} else {
+			console.warn(
```

Comment:
I am not sure if we should do it like that. I think you are waiting for our own logger, are you? Let's talk about that later, please.